src/pages/UserPosts/Explore.jsx

```diff
@@ -1,16 +1,20 @@
 import { data } from "../../helper/data";
 import { Posts } from "../../Components";
 import "./user-posts.css";
+import { Navigation } from "../../Components";
 
 const Explore = () => {
   return (
-    <div className="explore__container">
-      <div className="grid explore__grid">
-        {data.map((item, id) => (
-          <Posts post={item} key={id} />
-        ))}
+    <>
+      <Navigation />
+      <div className="explore__container">
+        <div className="grid explore__grid">
+          {data.map((item, id) => (
+            <Posts post={item} key={id} />
```

Comment:
Here I think you're giving the index as the key, which is anyway used as the default key. Better to use the post id for key.

Reply:
Currently used a dummy data for the UI part will make a note of this comment and incorporate it in the next pr